pkg/sdn/plugin/node.go

```diff
@@ -204,9 +202,23 @@ func (node *OsdnNode) Start() error {
 		}
 	}
 
-	// Wait for kubelet to init the plugin so we get a knetwork.Host
-	log.V(5).Infof("Waiting for kubelet network plugin initialization")
-	<-node.kubeletInitReady
```

Comment:
restore the channel wait. if we additionally want to poll until node.host.GetRuntime() returns non-nil, we can do that additionally